content/java/concepts/array-list/terms/set/set.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.set()'
+Description: 'Replaces the element present in a specified position in an ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.set()`** method is used for replacing the element present in a specified position with another element in instances of the `ArrayList` class. After execution, returns the replaced element's value.
+
+## Syntax
+
+An element can be replaced in an `ArrayList` instance with another element by passing new value to the `.set()` method with a specified index:
+
+```pseudo
+arrayListInstance.set(index, element);
+```
+
```

Comment:
```suggestion

An element at a specified `index` can be replaced in an `arrayListInstance` with the `newElement`.

> **Note:** The `newElement` must be of the same [data type](https://www.codecademy.com/resources/docs/java/data-types) as the rest of the elements in `arrayListInstance`. Otherwise, an error will occur.

```